### PR TITLE
added support for myupsmash command in the form of new whine model

### PR DESCRIPTION
**Pull request by coltatgh**
Created whines model to support the !myupsmash command for SmashNightBot

### CODE REVIEW

File: sncrs/data/admin.py [hunk 1]
```diff
@@ -23,6 +23,9 @@ class AliasInline(admin.TabularInline):
 class GreetingInline(admin.TabularInline):
     model = Greeting
 
+class WhineInline(admin.TabularInline):
```
Comment (Jyckle) [inline sncrs/data/admin.py:26]: Inline admin interfaces are great for editing a model from a related model! In this case, if the goal is to be able to add Whines from the Person admin interface, you need to add `WhineInline` to the list from line 33-37
Alternatively, if you don't intended for that to be the case, I would suggest removing lines 26-28

**Review by Jyckle — CHANGES_REQUESTED** [review]
Besides the one change on the admin interface, looks great!

File: sncrs/data/admin.py (hunk 1 above)
Comment (coltatgh) [reply sncrs/data/admin.py:26]: Ahhhh OK, yeah I had forgotten what the *InLines were used for. Thanks, I added it to PersonAdmin

**Review by Jyckle — APPROVED** [review]
Looks good! Thanks for making that change!

**Pull request merged**